NHS.Api/README.md

```diff
@@ -0,0 +1,46 @@
+DJANGO REST FRAMEWORK - RIAK INTEGRATION
+
+INTRODUCTION :-
+This application exposes django rest api for interaction with RIAK
+
+INSTALLATION :-
+Run requirement27.txt file.
+
+API's :-
+Trust CRUD :-
+Search : "http://<Domain Name>:<port_number>/search_trust/<trust_key_name>/"
+# Write : "http://<Domain Name>:<port_number>/write_trust/<trust_key_name>/"
+# Delete : "http://<Domain Name>:<port_number>/delete_trust/<trust_key_name>/"
+# Update : "http://<Domain Name>:<port_number>/update_trust/<trust_key_name>/"
+
+Region CRUD :-
+# Write : "http://<Domain Name>:<port_number>/write_region/<region_key_name>/"
+# Delete : "http://<Domain Name>:<port_number>/delete_region/<region_key_name>/"
+# Update : "http://<Domain Name>:<port_number>/update_region/<region_key_name>/"
+
+**Note :- Update API is not functional till now. Key_name(Name of Trust / Name of Region).
+
+Mock Data :
+The schema of Mock data is temporary and can be changed. For time being I have hard coded the data to be written in RIAK.
+
+TrustData = {
```

Comment:
We are storing the gap between the events,

So it should be 
TrustData = {
    "E1": 20, (Difference in days between E1 and E2 start time ) 
    "E2": 100,
    "E3": 25,
    "E4": 32
}
same will be followed for rest and for RegionData  as well